### PR TITLE
added some packages, services and people

### PR DESCRIPTION
**Pull request by guncebektas**
add: some packages, services and people
feat: index

### CODE REVIEW

**Comment by guncebektas** [comment]
I mean this one @jankapunkt

**Comment by jankapunkt** [comment]
```shell
  readme.md:7:1
  ✖    1:1    Git repository must be at least 30 days old                   remark-lint:awesome-git-repo-age
  ✖    7:1    Main heading must be in title case                            remark-lint:awesome-heading
  ✖   30:1    ToC item "News" link "#News" not found                        remark-lint:awesome-toc
  ✖  106:140  Text "NodeJS" should be written as "Node.js"                  remark-lint:awesome-spell-check
  ✖  112:53   List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  116:68   List item description must end with proper punctuation        remark-lint:awesome-list-item
  ✖  118:77   List item description must end with proper punctuation        remark-lint:awesome-list-item
  ✖  125:4    Text "Youtube" should be written as "YouTube"                 remark-lint:awesome-spell-check
  ✖  125:51   Text "Youtube" should be written as "YouTube"                 remark-lint:awesome-spell-check
```

@StorytellerCZ can we disable or ignore rule 1?
@guncebektas can you fix the other lint issues?

File: readme.md [hunk 1]
```diff
@@ -13,7 +14,7 @@ Awesome packages, articles, tips and people all around Meteor.js
 <!-- image -->
 
 <a href="" target="_blank" rel="noopener noreferrer">
-  <img src="" />
+  <img src="https://github.com/guncebektas/awesome-meteor/blob/main/awesome-meteor.jpg" width="200"/>
```
Comment (jankapunkt) [inline readme.md]: This should be `https://github.com/Meteor-Community-Packages/awesome-meteor/blob/main/awesome-meteor.jpg`

**Review by jankapunkt — CHANGES_REQUESTED** [review]

**Review by jankapunkt — APPROVED** [review]

**Comment by jankapunkt** [comment]
Approved, the one CI check fails due to deprecated deps in the ci config (related PR: #3). @StorytellerCZ can be merged from my end

File: readme.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
+<!--lint disable awesome-git-repo-age-->
 <div align="center">
 
 <!-- title -->
 
 <!--lint ignore no-dead-urls-->
 
-# Awesome Meteor.js [![Awesome](https://awesome.re/badge.svg)](https://awesome.re) [![lint](https://github.com/Meteor-Community-Packages/awesome-meteor/actions/workflows/lint.yaml/badge.svg)](https://github.com/Meteor-Community-Packages/awesome-meteor/actions/workflows/lint.yaml)
+# Awesome Meteor [![Awesome](https://awesome.re/badge.svg)](https://awesome.re) [![lint](https://github.com/Meteor-Community-Packages/awesome-meteor/actions/workflows/lint.yaml/badge.svg)](https://github.com/Meteor-Community-Packages/awesome-meteor/actions/workflows/lint.yaml)
```
Comment (StorytellerCZ) [inline readme.md:8]: It should be Meteor.js

File: readme.md [hunk 3]
```diff
@@ -27,35 +28,125 @@ Awesome packages, articles, tips and people all around Meteor.js
...
-- [Meteor 3 PR](https://github.com/meteor/meteor/pull/12359) - Development of the next generation of Meteor.js
+- [Meteor 2.13.1](https://docs.meteor.com/changelog#v213120230904) - Current recommended release.
+- [Meteor 3 PR](https://github.com/meteor/meteor/pull/12359) - Development of the next generation of Meteor.js.
 
-## News
+## News and Blogs
 
 - [Official Meteor Blog](https://blog.meteor.com/)
-- [Meteor Dispatched Weekly news podcast](https://www.youtube.com/@meteorjscommunity/podcasts)
+- [Meteor Dispatched - Weekly News Podcast](https://www.youtube.com/@meteorjscommunity/podcasts)
```
Comment (StorytellerCZ) [inline readme.md]: Why is every word capitalized? It is not name of business or anything else.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

**Comment by StorytellerCZ** [comment]
Merging, but will fix the grammar issue in commit later.

File: readme.md (hunk 2 above)
Comment (guncebektas) [reply readme.md:8]: linter warns about title case when .js

File: readme.md (hunk 3 above)
Comment (guncebektas) [reply readme.md]: you are right. let's remove that weekly part?

**Pull request merged**